app/app.py

```diff
@@ -34,16 +32,16 @@ def test_pipeline(
     thr_mask: float = 0.5,
 ):
     # set and save parameters
-    parameters0 = voluseg.parameter_dictionary()
+    parameters0 = voluseg.get_parameters_dictionary()
```

Comment:
This seems like a temporary change, correct?
We should not change the name since it will break backwards compatibility.

Reply:
@mikarubi I changed that in the last PR, felt like a better naming for this function:
https://github.com/mikarubi/voluseg/blob/3b8465d710815eac7733d681880a67eef891bc73/voluseg/_tools/parameter_dictionary.py#L5

but of course, if you want I can quickly revert that to the original naming

Reply:
Let's revert to avoid needless backward compatibility issues. Eventually, this function may not be necessary at all, since all the parameters and processing could start at step0. In this case, we can just make it issue a warning to the users.